### DISCUSSION TITLE
Why is the proxy delay large

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#27776] [Q&A and General discussion]
milvus_proxy_sq_wait_result_latency_aihfki  and  milvus_proxy_sq_latency_aihfki    and  milvus_proxy_sq_latency_aihfki    delay large，
What is the cause

**Comment by yhmo** [comment]
proxy_sq_wait_result_latency and milvus_proxy_sq_latency is the time cost from the proxy sending a search request to query nodes to receive results from query nodes. 

If the collection is huge, the latency could be more than a second.
![Screenshot from 2023-10-19 10-26-31](https://github.com/milvus-io/milvus/assets/2282099/75ad382e-5f0c-475b-9046-f58164850c97)

**Comment by 1292253144** [comment]
V2.2.8，

**Reply by 1292253144** [reply]
![Uploading image.png…]()
